dts/riscv/starfive/jh7110-visionfive-v2.dtsi

```diff
@@ -0,0 +1,211 @@
+			};
+		};
+
+		U74_3: cpu@3 {
+			compatible = "sifive,u74", "riscv";
+			device_type = "cpu";
+			d-cache-block-size = <64>;
+			d-cache-sets = <64>;
+			d-cache-size = <32768>;
+			d-tlb-sets = <1>;
+			d-tlb-size = <40>;
+			i-cache-block-size = <64>;
+			i-cache-sets = <64>;
+			i-cache-size = <32768>;
+			i-tlb-sets = <1>;
+			i-tlb-size = <40>;
+			mmu-type = "riscv,sv39";
+			next-level-cache = <&ccache>;
+			reg = <0x3>;
+			riscv,isa = "rv64imafdcg";
+			tlb-split;
+			cpu3_intc: interrupt-controller {
+				compatible = "riscv,cpu-intc";
+				#interrupt-cells = <1>;
+				interrupt-controller;
+			};
+		};
+
+		U74_4: cpu@4 {
+			compatible = "sifive,u74", "riscv";
+			device_type = "cpu";
+			d-cache-block-size = <64>;
+			d-cache-sets = <64>;
+			d-cache-size = <32768>;
+			d-tlb-sets = <1>;
+			d-tlb-size = <40>;
+			i-cache-block-size = <64>;
+			i-cache-sets = <64>;
+			i-cache-size = <32768>;
+			i-tlb-sets = <1>;
+			i-tlb-size = <40>;
+			mmu-type = "riscv,sv39";
+			next-level-cache = <&ccache>;
+			reg = <0x4>;
+			riscv,isa = "rv64imafdcg";
+			tlb-split;
+			cpu4_intc: interrupt-controller {
+				compatible = "riscv,cpu-intc";
+				#interrupt-cells = <1>;
+				interrupt-controller;
+			};
+		};
+	};
+
+	ram0: memory@8000000 {
+		device_type = "memory";
+		reg = <0x0 0x8000000 0x1 0x0>;
+	};
+
+	soc {
+		#address-cells = <2>;
+		#size-cells = <2>;
+		#clock-cells = <1>;
+		compatible = "starfive,jh7110", "simple-bus";
+		ranges;
+
+		clint: timer@2000000 {
+			compatible = "starfive,jh7110-clint", "sifive,clint0";
+			interrupts-extended = <&cpu0_intc 3 &cpu0_intc 7
+						&cpu1_intc 3 &cpu1_intc 7
+						&cpu2_intc 3 &cpu2_intc 7
+						&cpu3_intc 3 &cpu3_intc 7
+						&cpu4_intc 3 &cpu4_intc 7>;
+			reg = <0x0 0x2000000 0x0 0x10000>;
+		};
+
+		ccache: cache-controller@2010000 {
+			cache-block-size = <64>;
+			cache-level = <2>;
+			cache-sets = <2048>;
+			cache-size = <2097152>;
+			cache-unified;
+			compatible = "starfive,jh7110-ccache", "sifive,ccache0", "cache";
+			interrupt-parent = <&plic>;
+			interrupts = <1>, <3>, <4>, <2>;
+			reg = <0x0 0x2010000 0x0 0x4000>;
+		};
+
+		plic: interrupt-controller@c000000 {
+			compatible = "starfive,jh7110-plic", "sifive,plic-1.0.0";
+			#address-cells = <0>;
+			#interrupt-cells = <2>;
+			interrupt-controller;
+			interrupts-extended = <&cpu0_intc 11>,
+						<&cpu1_intc 11>, <&cpu1_intc 9>,
+						<&cpu2_intc 11>, <&cpu2_intc 9>,
+						<&cpu3_intc 11>, <&cpu3_intc 9>,
+						<&cpu4_intc 11>, <&cpu4_intc 9>;
+			reg = <0x0 0x0c000000 0x0 0x04000000>;
+			riscv,max-priority = <7>;
+			riscv,ndev = <52>;
+		};
+
+		uart0: serial@10000000 {
+			compatible = "ns16550", "snps,dw-apb-uart";
+			clocks = <&uartclk>, <&apb2clk>;
+			clock-names = "baudclk", "apb_pclk";
+			clock-frequency = <100000000>;
+			current-speed = <115200>;
+			interrupt-parent = <&plic>;
+			interrupts = <32 1>;
+			reg = <0x0 0x10000000 0x0 0x10000>;
+			reg-shift = <2>;
+			status = "okay";
```

Comment:
`uart0` and `uart1` should be selected in the board dts, unless we're 100% sure that they need to be used for this SoC.

Reply:
uart0 and uart1 disabled from dtsi and enabled from dts.